src/ui-router-tabs.js

```diff
@@ -99,9 +102,9 @@ angular.module('ui.router.tabs').directive(
 }]
 ).run(
 ['$templateCache', function($templateCache) {
-    var DEFAULT_TEMPLATE = '<div><uib-tabset class="tab-container" type="{{type}}" vertical="{{vertical}}" ' +
+    var DEFAULT_TEMPLATE = '<div><uib-tabset class="tab-container" active="activeTab" type="{{type}}" vertical="{{vertical}}" ' +
       'justified="{{justified}}">' + '<uib-tab class="tab" ng-repeat="tab in tabs" heading="{{tab.heading}}" ' +
-      'active="tab.active" disable="tab.disable" ng-click="go(tab)">' +
+      'disable="tab.disable" index="$index" ng-click="go(tab)">' +
```

Comment:
Hmm, well, there are a few releases in 1.2.x where the tab index is required. For the default template, it shouldn't matter at all since there is no custom filters or anything special being done here.